lesson-17/index.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+
+// HW-17
+
+// task 1
+
+// const a = ("'true'");
```

Comment:
Зачем ковычки и скобки? Можно упросить
```suggestion
// const a = 'true';
```